src/AlertBox/AlertBox.less

```diff
@@ -4,22 +4,25 @@
   .margin--y--m;
   .padding--s;
   .border--s(transparent);
-  border-left-width: @size_xs;
 }
 
 .AlertBox--title {
-  .margin--left--xs;
   .text--semi-bold;
   .text--large;
+  .margin--bottom--s();
```

Comment:
perhaps out of scope of this PR, but should we add `()` to the rest of the mixins in this file to stay consistent? esp. since mixins without parentheses is deprecated

Reply:
That's a good point. I'll change the others for consistency.